docs/variables/countdown.md

```diff
@@ -4,33 +4,51 @@ id: countdown
 
 # $(countdown)
 
-Countdown tells you the amount of time until a certain date has passed.
+Returns the time that is left until a certain date has passed. Shares this behaviour with `$(countup)`.
```

Comment:
Imo it should be clarified that as it shares this behaviour, it can count both up to, and from a time, once the time passes, it will be a countup. If it's before the time has been reached, it's a countdown.

Reply:
I'll add a new entry for `$(countup)` and change the description of `$(countdown)` to reflect the equal behavior in a different way.